backend/lcfs/web/api/fuel_code/repo.py

```diff
@@ -53,30 +47,17 @@ async def get_fuel_types(self) -> List[FuelType]:
     @repo_handler
     async def get_formatted_fuel_types(self) -> List[Dict[str, Any]]:
         """Get all fuel type options with their associated fuel categories and fuel codes"""
-        # Define the filtering conditions for fuel codes
```

Comment:
I read the comments but didn't see a mention why this is being removed. Is this not working correctly or is there validation for expired fuel codes happening elsewhere?